### ISSUE TITLE
[Bug] Integration not working after update to 2.0.27

### ISSUE THREAD

**Issue by goldsam328** [greghesp/ha-bambulab#675]
### Describe the bug

After upgrading to version 2.0.27, the integration has stopped connecting to my printer. I have tried to reload the integration but it doesn’t seem to fix it. Almost all sensors show as unavailable or offline and I’m unable to control things like the speed and the light. The AMS also honest work as it shows that the entity is no longer being used by this integration. 
![image](https://github.com/user-attachments/assets/59d71abe-bf63-41ce-8414-89ee6c7d9200)
![image](https://github.com/user-attachments/assets/58487d0a-db7b-42d1-b015-565b6180fddb)

### To Reproduce

This issue arose after updating to the latest version, 2.0.27.

### Expected Behaviour

The integration to continue working as it was before the upgrade.

### What device are you using?

A1

### Diagnostic Output

```shell
{
  "home_assistant": {
    "installation_type": "Home Assistant OS",
    "version": "2024.10.4",
    "dev": false,
    "hassio": true,
    "virtualenv": false,
    "python_version": "3.12.4",
    "docker": true,
    "arch": "x86_64",
    "timezone": "America/New_York",
    "os_name": "Linux",
    "os_version": "6.6.46-haos",
    "supervisor": "2024.10.3",
    "host_os": "Home Assistant OS 13.1",
    "docker_version": "26.1.4",
    "chassis": "vm",
    "run_as_root": true
  },
  "custom_components": {
    "hacs": {
      "documentation": "https://hacs.xyz/docs/configuration/start",
      "version": "2.0.1",
      "requirements": [
        "aiogithubapi>=22.10.1"
      ]
    },
    "kwikset": {
      "documentation": "https://github.com/explosivo22/kwikset-ha/",
      "version": "0.3.0",
      "requirements": [
        "aiokwikset==0.3.0a7"
      ]
    },
    "proxmoxve": {
      "documentation": "https://github.com/dougiteixeira/proxmoxve",
      "version": "3.4.4",
      "requirements": [
        "proxmoxer==2.0.1"
      ]
    },
    "adaptive_lighting": {
      "documentation": "https://github.com/basnijholt/adaptive-lighting#readme",
      "version": "1.22.0",
      "requirements": [
        "ulid-transform"
      ]
    },
    "bambu_lab": {
      "documentation": "https://github.com/greghesp/ha-bambulab",
      "version": "2.0.27",
      "requirements": [
        "httpx>=0.27.0"
      ]
    },
    "fontawesome": {
      "documentation": "https://github.com/thomasloven/hass-fontawesome",
      "version": "2.2.3",
      "requirements": []
    }
  },
  "integration_manifest": {
    "domain": "bambu_lab",
    "name": "Bambu Lab",
    "codeowners": [
      "greghesp",
      "AdrianGarside"
    ],
    "config_flow": true,
    "dependencies": [
      "device_automation",
      "ffmpeg",
      "mqtt"
    ],
    "documentation": "https://github.com/greghesp/ha-bambulab",
    "iot_class": "local_push",
    "issue_tracker": "https://github.com/greghesp/ha-bambulab/issues",
    "requirements": [
      "httpx>=0.27.0"
    ],
    "ssdp": [
      {
        "st": "urn:bambulab-com:device:3dprinter:1"
      }
    ],
    "version": "2.0.27",
    "is_built_in": false
  },
  "setup_times": {
    "null": {
      "setup": 4.668103065341711e-05
    },
    "01JBJP6C34HBW68Z5WQD4PADP7": {
      "wait_import_platforms": -0.0738525209017098,
      "wait_base_component": -0.000830034026876092,
      "config_entry_setup": 0.0963295359397307
    }
  },
  "data": {
    "config_entry": {
      "created_at": "2024-11-01T01:42:15.908478+00:00",
      "data": {
        "device_type": "A1",
        "serial": "**REDACTED**"
      },
      "discovery_keys": {},
      "disabled_by": null,
      "domain": "bambu_lab",
      "entry_id": "01JBJP6C34HBW68Z5WQD4PADP7",
      "minor_version": 1,
      "modified_at": "2024-11-04T01:57:38.197751+00:00",
      "options": {
        "region": "NorthAmerica",
        "email": "**REDACTED**",
        "username": "**REDACTED**",
        "name": "3DP-039-800",
        "host": "",
        "local_mqtt": false,
        "auth_token": "**REDACTED**",
        "access_code": "**REDACTED**",
        "usage_hours": 34.199999999999996
      },
      "pref_disable_new_entities": false,
      "pref_disable_polling": false,
      "source": "user",
      "title": "**REDACTED**",
      "unique_id": null,
      "version": 2
    },
    "push_all": null,
    "get_version": null
  }
}
```

### Log Extracts

```shell
024-11-04 12:13:26.270 WARNING (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] On Disconnect: Printer disconnected with error code: 0
2024-11-04 12:13:26.270 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] _on_disconnect: Lost connection to the printer
2024-11-04 12:13:26.271 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Stopping watchdog thread
2024-11-04 12:13:26.271 INFO (A1-Watchdog-1058) [custom_components.bambu_lab.pybambu] Watchdog thread exited.
2024-11-04 12:13:26.271 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Connect: Attempting Connection to us.mqtt.bambulab.com
2024-11-04 12:13:26.436 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Starting listen loop
2024-11-04 12:13:26.596 INFO (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] On Connect: Connected to printer
2024-11-04 12:13:26.596 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Loading slicer settings...
2024-11-04 12:13:26.596 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Loading slicer settings
2024-11-04 12:13:26.596 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Getting slicer settings from Bambu Cloud
2024-11-04 12:13:26.643 ERROR (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Slicer settings load failed: 403
2024-11-04 12:13:26.643 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Now subscribing...
2024-11-04 12:13:26.643 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Subscribing: device/03919D460402800/report
2024-11-04 12:13:26.643 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] On Connect: Getting version info
2024-11-04 12:13:26.643 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Sent {'info': {'sequence_id': '0', 'command': 'get_version'}} to topic device/03919D460402800/request
2024-11-04 12:13:26.643 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] On Connect: Request push all
2024-11-04 12:13:26.643 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Sent {'pushing': {'sequence_id': '0', 'command': 'pushall'}} to topic device/03919D460402800/request
2024-11-04 12:13:26.643 DEBUG (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Starting watchdog thread
2024-11-04 12:13:26.644 INFO (A1-Watchdog-1058) [custom_components.bambu_lab.pybambu] Watchdog thread started.
2024-11-04 12:13:26.645 ERROR (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] A listener loop thread exception occurred:
2024-11-04 12:13:26.645 ERROR (A1-Mqtt-67) [custom_components.bambu_lab.pybambu] Exception. Type: <class 'AttributeError'> Args: 'BambuClient' object has no attribute '_client'
```

**Comment by Datastore24Kirill** [comment]
I also have this problem after update. I can't connect

**Comment by jneilliii** [comment]
related to #673 possibly, looks like you both are cloud connected?

**Comment by goldsam328** [comment]
I am cloud connected so it may be related to #673.

**Comment by Datastore24Kirill** [comment]
Me too... I am cloud connected so it may be related to #673.

**Comment by NikoGrub** [comment]
I have received my A1 mini today and can't login either.

**Comment by AdrianGarside** [comment]
Fixed the error output with #676.
